packages/react-app/src/views/HomeView.jsx

```diff
@@ -1,17 +1,32 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
 import { Typography, Space } from "antd";
 import SignatureSignUp from "../components/SignatureSignUp";
+import { getAllEvents } from "../data/api";
+import { eventToString } from "../helpers/events";
 
 const { Text, Title } = Typography;
 
 export default function HomeView({ serverUrl, address, userProvider }) {
+  const [eventsFeed, setEventFeeds] = useState([]);
+  useEffect(() => {
+    const updateEvents = async () => {
+      const events = await getAllEvents();
+      setEventFeeds(events);
+    };
+
+    updateEvents();
+  }, []);
+
   return (
     <div className="container">
       <Space direction="vertical">
         <Title>Welcome to scaffold-directory!</Title>
         <Text>Sign, build and show!</Text>
         <SignatureSignUp serverUrl={serverUrl} address={address} userProvider={userProvider} />
         <Title level={3}>Activity feed</Title>
+        {eventsFeed.map(event => (
+          <div>{eventToString(event)}</div>
```

Comment:
We might want to sort these by timestamp DESC. Maybe a param for `getAllEvents`? Or we could just sort it in the frontend

Reply:
yep, just added this before the demo yesterday. I'm pushing it now